account/src/main/java/com/caldevsupplychain/account/service/AccountService.java

```diff
@@ -0,0 +1,40 @@
+package com.caldevsupplychain.account.service;
+
+
+
+import com.caldevsupplychain.account.model.Company;
+import com.caldevsupplychain.account.model.User;
+import com.caldevsupplychain.common.ws.account.UserWS;
+
+import javax.swing.text.html.Option;
+import java.util.Optional;
+
+public interface AccountService {
+
+    boolean userExist(String emailAddress);
+
+    User createUser(UserWS userWS);
+
+    User updateUser(long id, String username, String emailAddress, String password, String role, Optional<String> companyName);
```

Comment:
Have one `UserBean updateUser(UserBean userBean` and move update username/password/company all into one method.

Reply:
Fixed updated user. BUT apache shiro permission does not work even though I already setup the required part. Need to be review again